routers/web/repo/repo.go

```diff
@@ -590,23 +590,25 @@ func SearchRepo(ctx *context.Context) {
 		return
 	}
 
+	// To improve performance when only the count is requested
+	if ctx.FormBool("count_only") {
+		ctx.SetTotalCountHeader(count)
```

Comment:
As that header will always be set (see line 614), what about moving it prior to the if and deleting the mentioned lines?